modules/cli/src/main/scala/scala/cli/commands/Version.scala

```diff
@@ -4,11 +4,47 @@ import caseapp._
 
 import scala.build.internal.Constants
 import scala.cli.CurrentParams
+import scala.cli.internal.ProcUtil
 
-object Version extends ScalaCommand[VersionOptions] {
+class Version(appName: String) extends ScalaCommand[VersionOptions] {
```

Comment:
```suggestion
object Version extends ScalaCommand[VersionOptions] {
```
`appName` is unused in `Version` class.

Reply:
Yep, I’m gonna fix it!